src/App.tsx

```diff
@@ -1,50 +1,111 @@
-import React from 'react';
 import 'bulma/css/bulma.css';
+import { useState } from 'react';
+import classNames from 'classnames';
+
 import './App.scss';
+import { goodsFromServer } from './api/goodsFromServer';
+import { SortType } from './enums/sortingEnum';
+
+export const App = () => {
+  const [goods, setGoods] = useState(goodsFromServer);
+  const [activeSort, setActiveSort] = useState<SortType | ''>('');
+  const [isReversed, setIsReversed] = useState(false);
+
+  const sortGoods = (compareFunction: (a: string, b: string) => number) => {
+    const sortedGoods = [...goods].sort(compareFunction);
+
+    setGoods(sortedGoods);
+  };
```

Comment:
According to the checklist, direct DOM interactions should be avoided and React should be used as much as possible. The `sortGoods` function directly manipulates the state of the goods, which could lead to direct DOM updates. Consider using functional updates with `setGoods` to ensure that you're not directly mutating state.